packages/synthetix-main/cannonfile.toml

```diff
@@ -276,25 +252,6 @@ factory.SNXProxy.abiOf = ["SNXRouter"]
 factory.SNXProxy.event = "AssociatedSystemSet"
 factory.SNXProxy.arg = 2
 
-[invoke.init_esnx]
-target = ["CoreProxy"]
-from = "<%= settings.owner %>"
-func = "initOrUpgradeToken"
-args = [
-  "0x65534e58546f6b656e0000000000000000000000000000000000000000000000",
-  "Escrowed Synthetix",
-  "eSNX",
-  18,
-  "<%= contracts.ESNXRouter.address %>"
-]
-depends = [
-  "invoke.upgrade_core_proxy",
-  "contract.ESNXRouter",
-]
-factory.ESNXProxy.abiOf = ["ESNXRouter"]
-factory.ESNXProxy.event = "AssociatedSystemSet"
-factory.ESNXProxy.arg = 2
```

Comment:
You need to remove the ESNX reference in the RewardDistributorModule.test as well